src/kili/services/plugins/tools.py

```diff
@@ -0,0 +1,13 @@
+"""
+Set of common functions used by plugins
+"""
+
+DUPLICATE_ERROR_SUBSTRING = "an entity Plugin already exists with value"
+
+def check_errors_plugin_upload(result, file_path, plugin_name):
+    """
+    Check if the error during upload is because the plugin already
+    exists and print the upload function for the user
+    """
+    if "errors" in result and DUPLICATE_ERROR_SUBSTRING in result["errors"][0]["message"]:
+        print('Perhaps you wanted to use update_plugin("%s", "%s") (Warning, the existing plugin will be discarded)' % (file_path, plugin_name))
```

Comment:
And you should use the logger as @gheorghetutunaru did in his PR with fstrings

Reply:
If we see that there is this error, shouldn't we throw an Exception instead of just logging ? Because now we will have this log, but also all the graphql error being printed, which is kind of redundant